source/guides/stacks/dlrs-inference.rst

```diff
@@ -0,0 +1,1035 @@
+.. _dlrs-inference:
+
+AI Inference with the Deep Learning Reference Stack
+###################################################
+
+In this guide we explore a solution for using the Deep Learning Reference Stack with a Seldon Core\* platform that has been deployed on Kubernetes\*.
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+The solution covered here uses the following software components:
+
```

Comment:
I think we're missing an opportunity here.  The Overview lacks a clear explanation of *how* all of these components are used in concert. A simple, succinct sentence will solve this like a shot to the heart. I recommend adding that "goal" statement here--relating how all these components fit together. Also, I'd like to see these components as short, uniform descriptions.  I think what's missing here is the "keystone" sentence at the beginning (above); then each one of these descriptions can explain how it serves that goal.

Reply:
@bd-dean. BTW:  For comparison, I like the[ intro sentences in the Overview of the main DLRS guide.](https://docs.01.org/clearlinux/latest/guides/stacks/dlrs.html#overview) Maybe you could write something similar here.

Reply:
What do you think? 
IMHO: If this solution requires all of these components, it should use "requires" instead of "uses".  I'm hoping there could be a simple, direct sentence without nuance.